source/stylesheets/base/_application-header.scss

```diff
@@ -0,0 +1,33 @@
+.main-nav {
+	display: flex;
+	padding: $small-spacing $base-spacing;
+}
+
+@media only screen and (max-width : $small-breakpoint) {
```

Comment:
`@mixin breakpoint($property)  {
    @media only screen and (min-width : $property) {
        @content;
    }
}`

Here's my mixin, seems to be working - any feedback on it?

Reply:
I structured my css to be mobile first - seemed to add some additional lines of code since I had to set flex-direction to column, and then the row for medium+ width. Still the best method even though a lot of my css resides within a media query?